Author: Jordi Mallach <jordi@debian.org>
Description: Check for GNU.
 Fix builds on hurd-i386 by checking for __GNU__.
Forwarded: no

Index: mess-0.144/src/osd/sdl/sdlprefix.h
===================================================================
--- mess-0.144.orig/src/osd/sdl/sdlprefix.h	2012-01-15 23:05:38.165375833 +0100
+++ mess-0.144/src/osd/sdl/sdlprefix.h	2012-01-15 23:05:57.725376275 +0100
@@ -42,7 +42,7 @@
 /* Large file support on IRIX needs _SGI_SOURCE */
 #undef _POSIX_SOURCE
 
-#elif defined(__linux__) || defined(__FreeBSD_kernel__)
+#elif defined(__linux__) || defined(__GNU__) || defined(__FreeBSD_kernel__)
 #define SDLMAME_LINUX 1
 
 #elif defined(__FreeBSD__)
